apps/docs/src/app/platform/component-docs/platform-forms/platform-datetime-picker/platform-datetime-picker-examples/platform-datetime-picker-reactive-example.component.ts

```diff
@@ -0,0 +1,47 @@
+import { ChangeDetectionStrategy, Component, ViewEncapsulation } from '@angular/core';
+import { FormGroup, ValidatorFn, Validators } from '@angular/forms';
+import { FdDate, FdDatetime } from '@fundamental-ngx/core';
+
+@Component({
+    selector: 'fdp-datetime-picker-reactive-example',
+    templateUrl: './platform-datetime-picker-reactive-example.component.html',
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class PlatformDatetimePickerReactiveExampleComponent {
+    storedDate: FdDatetime = FdDatetime.getToday();
+    date = FdDatetime.getToday();
+
+    public datetimePickerForm = new FormGroup({});
+    requiredDateValidator: ValidatorFn[];
+
+    data: StoredDatetimeObject;
+
+    constructor() {
+        this.storedDate = new FdDatetime(new FdDate(2008, 2, 11), this.storedDate.time);
+        this.data = new StoredDatetimeObject(this.storedDate);
+        this.requiredDateValidator = [Validators.required];
```

Comment:
you can init in the initialization of method

```
storedDate: FdDatetime = new FdDatetime(new FdDate(2008, 2, 11), this.storedDate.time);
date: Date = new StoredDatetimeObject(this.storedDate);
requiredDateValidator: ValidatorFn[] = [Validators.required];
```

Reply:
can change for `requiredDateValidator` but for `storedDate` since it uses time from `storedDate` itself, it will complain of usage before initialization, so it would need some dummy value to be initalized. Similarly for `date`, it would complain of not having all the fields of `Date` or `FdDatetime` type and also I would like it to take the `2008, 2, 11` value of `storedDate` so I would prefer having it in the constructor.